cmd/gosec/main.go

```diff
@@ -165,19 +165,19 @@ func loadRules(include, exclude string) rules.RuleList {
 	return rules.Generate(filters...)
 }
 
-func saveOutput(filename, format string, issues []*gosec.Issue, metrics *gosec.Metrics, errors map[string][]gosec.Error) error {
+func saveOutput(filename, format, rootPath string, issues []*gosec.Issue, metrics *gosec.Metrics, errors map[string][]gosec.Error) error {
```

Comment:
I'm not sure modifying the interface to achieve a truncated file path is the best solution here. Is there any specific reason for it? Sonarqube only recognize relative paths?

This is certainly something we can look at fixing though.

Reply:
Yes, sonarqube reports from the root of the project. I needed a way to remove the full path.  Happy to refactor, if you can think of a better way of doing this.

Reply:
Example of this working:
![image](https://user-images.githubusercontent.com/40531/54163137-58c4fd00-442e-11e9-9b06-9518ed4a673e.png)